bentoctl/deployment_config.py

```diff
@@ -20,15 +20,23 @@
 local_operator_registry = get_local_operator_registry()
 
 
-def get_bento_path(bento: t.Union[str, Path]):
+def get_bento_path(bento_name_or_path: t.Union[str, Path]):
     try:
```

Comment:
1: if bento_name_or_path is dir and if (bento_name_or_path, bento.yaml) is a file

2: bentoml.get